services/QuillLMS/app/models/feedback_history.rb

```diff
@@ -221,6 +221,21 @@ def self.completeness_filter_query(complete)
     SQL
   end
 
+  def self.responses_for_scoring
```

Comment:
It's not clear to me how this code maps to the spec. 

`all prompts sum to > 5 || each prompt has at least one entry` ?

Reply:
If you end up keeping this nested case structure, I recommend breaking out the two substatements into separate ruby methods, then combine them through string interpolation, so that they're easier to interpret.

Reply:
@happythenewsad apologies, the logic for this was communicated over Slack with Emma. The requested logic for responses for scoring is: 6+ responses per session OR sessions with 2+ responses for each conjunction, just added that as a comment in the Notion card

Reply:
@happythenewsad just pushed changes to break that statement out into two substatements